chapters/synchronous.tex

```diff
@@ -127,30 +122,32 @@ \subsection{Rationale for Clocked Semantics}\label{rationale-for-clocked-semanti
     controller part, and different integer multiples are given, then the
     translator has to accept this (no error is detected).
   \end{enumerate}
+  Note: Clocked systems can mix different sampling rates
+  in well-defined ways when needed.
 \item
-  Due to the automatic sample and hold semantics, all variables
-  assigned in a when-clause of the above kind must have an initial value
+  Fewer initial conditions are needed, as only a subset of clocked
+  variables need initial conditions - the clocked state variables \cref{clocked-state-variables}.
+  For a standard when-clause all variables
+  assigned in a when-clause must have an initial value
   because they might be used, before they are assigned a value the first
   time. As a result, all these variables are ``discrete-time states''
-  although in reality only a small subset of them need an initial
+  although in reality only a subset of them need an initial
   value.
 \item
-  Only a restricted form of equations can be used in a standard
-  when-clause, since the left hand side has to be a variable, in order
-  to identify the variables that are assigned in the when-clause. This
-  is a severe restriction, especially if nonlinear control algorithms
-  shall be defined. This restriction is not present for clocked
-  equations.
+  More general equations can be used, compared to standard when-clauses that require
+  a restricted form of equations where the left hand side has to be a variable, in order
+  to identify the variables that are assigned in the when-clause.
```

Comment:
Use parenthesis instead of comma to make clear what the last part refers to:
```suggestion
  a restricted form of equations where the left hand side has to be a variable (in order
  to identify the variables that are assigned in the when-clause).
```